README.md

```diff
@@ -1,2 +1,58 @@
 # CC.ui
 Collaborative Code UI.
+mockup https://www.figma.com/file/woKRnr0V6wEzOsCTu3pA0U/Live-Coding-App?node-id=1%3A3
+
+description https://docs.google.com/document/d/1nRaXjmNPGJMoNiC37SxKDyMah8exJpWiYUTCvTsESsM/edit
+
+questions sources: 
+https://github.com/learning-zone/css-basics?ref=opensource  https://github.com/sudheerj/reactjs-interview-questions  https://github.com/sudheerj/javascript-interview-questions#how-to-invoke-an-iife-without-any-extra-brackets https://github.com/Devinterview-io/react-native-interview-questions https://github.com/huynq-tp/nodejs-interview-questions https://github.com/kansiris/SQL-interview-questions https://github.com/learning-zone/docker-and-kubernetes-basics  https://github.com/Devinterview-io/redis-interview-questions
+
+
+Technology stack: 
+* TypeScript
+* NextJS
+* Redux-Toolkit
+* React-Router-dom
+* Material UI
+* HTML
+* CSS
+
+NodeJS version 15.0 and higher
+
+About:
+Quiz for online training  using the question-answer system
+with the ability to create your own training modules.
+
+How you can start or test project: 
+
+1. npm: 
```

Comment:
I suggest sticking with yarn, as using both may lead to issues in lock files which causes inconsistencies in production CI/CD

https://stackoverflow.com/questions/49589493/is-there-any-harm-in-using-npm-and-yarn-in-the-same-project